addons/modules/functions/fnc_moduleHideTerrainObjects.sqf

```diff
@@ -26,14 +26,14 @@ deleteVehicle _logic;
         "str_3den_garbagecollection_attribute_mode_displayname",
         [true, 1, 2, ["STR_Disp_Show", "STR_Disp_Hide"]]
     ],
-    ["EDIT", ELSTRING(common,Radius_Units), 10],
+    ["SLIDER", ELSTRING(common,Radius_Units), [1, 100, 10, 0]],
```

Comment:
Maximum of 100 m seems very low,

Reply:
That's kind of hiding a small woods area or city blocks already. I'm certainly open to increasing it to a more useful value.

Reply:
I think 500-1000 meters would be nice.

Reply:
1000 m is probably a good maximum. We can increase it later if needed.